client/my-sites/theme/main.jsx

```diff
@@ -638,25 +637,18 @@ class ThemeSheet extends React.Component {
 		}
 
 		let pageUpsellBanner, previewUpsellBanner;
-		const hasUpsellBanner =
-			! isJetpack &&
-			isPremium &&
-			! hasUnlimitedPremiumThemes &&
-			config.isEnabled( 'upsell/nudge-a-palooza' ) &&
-			abtest( 'themesNudgesUpdates' ) === 'test';
+		const hasUpsellBanner = ! isJetpack && isPremium && ! hasUnlimitedPremiumThemes;
 		if ( hasUpsellBanner ) {
-			// This is just for US-english audience and is not translated, remember to add translate() calls before
-			// removing a/b test check and enabling it for everyone
 			pageUpsellBanner = (
 				<Banner
 					plan={ PLAN_PREMIUM }
 					className="is-particular-theme-banner" // eslint-disable-line wpcalypso/jsx-classname-namespace
-					title={ 'Access this theme for FREE with a Premium or Business plan!' }
-					description={
+					title={ translate( 'Access this theme for FREE with a Premium or Business plan!' ) }
+					description={ translate(
 						'Instantly unlock all premium themes, more storage space, advanced customization, video support, and more when you upgrade.'
-					}
+					) }
 					event="themes_plan_particular_free_with_plan"
-					callToAction={ 'View plans' }
+					callToAction={ translate( 'View plans' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'View Plans' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).